content/intro-to-storybook/react/nl/simple-component.md

```diff
@@ -126,7 +126,7 @@ Zodra we dit hebben gedaan, zou het opnieuw opstarten van de Storybook-server te
 
 Nu we Storybook opgezet hebben, styles geīmporteerd en testcases hebben uitgebouwd, kunnen we snel beginnen met het implementeren van de HTML van de component om met het ontwerp overeen te komen.
 
-De component is momenteel nog basic. Schrijf eerst de code waarmee het ontwerp wordt bereikt zonder al te veel in detail te treden:
+De component is momenteel nog basaal. Schrijf eerst de code waarmee het ontwerp wordt bereikt zonder al te veel in detail te treden:
```

Comment:
Is this used commonly in the Netherlands? I've honestly never heard this word before

Reply:
It's not used that often, but it's the best word to describe the situation without falling back to English. It's mostly used as an adjective (e.g. "basale kennis") instead of as a noun. I'm fine with 'basic' too, although I prefer avoiding English words outside of technical terms.